tests/test_TFEngine.py

```diff
@@ -391,15 +391,6 @@ def test_engine_train_grad_noise_sparse():
   # You might see "Converting sparse IndexedSlices to a dense Tensor of unknown shape."
   # but that is normally related to sth else.
 
-  def wrapped_slice_to_tensor(value, dtype=None, name=None, as_ref=False):
-    print("wrapped_slice_to_tensor:", value)
-    #assert "flatten_with_seq_len_mask" in value.name
-    from tensorflow.python.ops import gradients_impl
-    return gradients_impl._IndexedSlicesToTensor(value, dtype=dtype, name=name, as_ref=as_ref)
-
-  from tensorflow.python.framework import ops
-  ops.register_tensor_conversion_function(tf.IndexedSlices, wrapped_slice_to_tensor, priority=1)
-
```

Comment:
Why only removing this? How does it fix sth?
Also, there was no problem in Travis. Why is this needed?

Reply:
They removed the `_IndexedSlicesToTensor` function from `gradients_impl.py`, I think they replaced it's functionality with something else. The test didn't pass when I checked locally, after the change it works fine.

Reply:
I see, ok.